trail/query/rest_views.py

```diff
@@ -1,24 +1,45 @@
 import django
 from django.core.exceptions import FieldError
 
-import numpy as np
-
 from rest_framework import mixins, status
-from rest_framework import exceptions as rf_error
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
 from upload.models import Metadata, Wcs
 from query.serializers import MetadataSerializer, WcsSerializer
 
-from drf_yasg.utils import swagger_auto_schema
-
 
 class GenericTableView(mixins.ListModelMixin, APIView):
+    """
+     GenericTableView that serves as an interface to provide one unify rest api query
+     to different tables
+    """
+
     queryset = None
     serializer_class = None
 
-    def get_shared_non_query_keys(self, queryDict, *args, **kwargs):
+    def get_shared_non_query_keys(self, queryDict):
+        """
+        Extracts the keys and values not directly related to the query from the given query dict,
```

Comment:
So if the first line is not longer than 120 chars required to trigger flake8 (https://github.com/dirac-institute/trailblazer/blob/main/setup.cfg#L5) why are the rest kept so short? Can you get them to look more like 120char wide block?